apps/meteor/client/views/room/providers/ComposerPopupProvider.tsx

```diff
@@ -64,14 +67,16 @@ const ComposerPopupProvider = ({ children, room }: ComposerPopupProviderProps) =
 					const filterRegex = filter && new RegExp(escapeRegExp(filter), 'i');
 					const items: ComposerBoxPopupUserProps[] = [];
 
-					const users = Array.from(usersFromRoomMessages.values())
-						.filter((u) => !!u.ts && (filterRegex ? u.username.match(filterRegex) || u.name?.match(filterRegex) : true))
```

Comment:
where did `!!u.ts` go?

Reply:
Asnwered above, since the Mongo query now only returns messages with the ts field this verification is unnecessary.